lib/test/unit/color-scheme.rb

```diff
@@ -5,9 +5,23 @@ module Unit
     class ColorScheme
       include Enumerable
 
+      TERM_256 = /
+        [+-]256color|
+        ^(alacritty|
+          iTerm\s?\d*\.app|
+          kitty|
+          mintty|
+          ms-terminal|
+          nsterm|
+          nsterm-build\d+|
+          terminator|
+          terminology(-[0-9.]+)?|
+          termite|
+          vscode)$/x
```

Comment:
Could you use `\z` instead of `$`?

Reply:
Could you sort this list in alphabetical order?

Reply:
Updated `\A` and `\z` and sorted.  IIRC, when I first started the regexp, I had sorted in alphabetical order, but the trailing `|` vs `)` had messed me up and after that I only sorted everything except the last value. :laughing:

Reply:
and actually, the list was already sorted alphabetically, if you ignore the trailing `|`. Should `nsterm|` sort before or after `nsterm-build\d+|`? My brain said yes. ASCII sorting says no. :slightly_smiling_face:

Reply:
Oh, sorry.